tests/e2e/support/commands.js

```diff
@@ -712,7 +712,11 @@ Cypress.Commands.add('playJackFromSevenOpponent', (card, target) => {
     .then((game) => {
       const player = game.players[game.myPNum];
       let foundCard;
-      const foundTarget = player.points.find((pointCard) => cardsMatch(target, pointCard));
+
+      const discarding = target === -1;
+      const foundTarget = discarding
+        ? -1
+        : player.points.find((pointCard) => cardsMatch(target, pointCard));
```

Comment:
Okay looking at it more closely, maybe instead of a ternary it'd make sense to use a regular conditional like this
```suggestion
      let targetId = -1;
      if (!discarding) {
        const foundTarget = player.points.find((pointCard) => cardsMatch(target, pointCard));
        if (!foundTarget) {
          throw new Error(
            `Error playing opponents jack: could not find ${target.rank} of ${target.suit} in player points`
          );
        }
        targetId = foundTarget.id
      }
```

Reply:
Do you think this is any more clear?. Then we could remove some of the stuff below

Reply:
I'll leave it to your judgement. What do you think is clearest?

Reply:
I think the ternary operator is sufficient here for the time being. Since you said you wanted to eventually rethink the -1 usage altogether, we could also revisit this then.